### DISCUSSION TITLE
Cannot see view

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#11269] [Help] [answered]
Hello all, created a view from one app users table.  Now the view exists because I can use it on Data Providers but, on Data, I cannot see the view, so I cannot delete it. Can you help me?

self-hosted   2.8.10

Thank you

**Comment by ConorWebb96** [comment, marked as answer]
Hey @JCV67,

I've replicated the issue you are experiencing and created a bug report here; https://github.com/Budibase/budibase/issues/11271. This should not be a difficult thing to fix as the functionality already exists for the other internal Budibase tables.

Thank you for highlighting this.
